### PR TITLE
React-icons-FontAwesome

### PR DESCRIPTION
**Pull request by howdyAnkit**
## Description
Exchanged all Existing Icons with React-Icon Font-Awesome

- [x] Font Awesome Icon
- [x] IconContext

## Changes in detail 

- [x] Changed the Existing icons from SVG to React-fonts (Font-awesome) Icon
- [x] Used yarn package named(Eeact-icons Refer:- https://www.npmjs.com/package/react-icons)
- [x] Made the Changes using React-Icons and IconContext
- [x] Font-awesome are applied via the simple yarn package (react-icons)
- [x] Used IconContext For styling and giving the tweaks to the Icons
- [x] Removed old previous SVG existing code
- [x] Used Props to get Color details from Theme.js 

## Dark Mode Before 
Screenshot (3)

## Dark Mode After
Screenshot (5)

## White Mode Before
Screenshot (6)
## White Mode After
Screenshot (7)

### CODE REVIEW

File: src/components/Icon.js [hunk 1]
```diff
@@ -2,71 +2,49 @@ import React from "react"
...
-    <svg
-      xmlns="http://www.w3.org/2000/svg"
-      width={size}
-      height={size}
-      viewBox="0 0 24 24"
-      aria-labelledby={name}
-      role="presentation"
-      className={className}
+    <IconContext.Provider
+      value={{ size: size, className: className, paddingTop: "2px" }}
```
Comment (samajammin) [inline src/components/Icon.js:31]: Why are we adding `paddingTop: "2px"` here?

**Comment by samajammin** [comment]
Thanks for this @howdyAnkit!

@ryancreatescopy curious to discuss with you how we'd like to approach this upgrade to a more scalable icon system. Should we find icons that closely resemble the current icons on the site? Or should we pick a library to roll with? We can also pick across available icon libraries on a case-by-case basis.

e.g. view the sun of the dark mode / light mode toggle

Current site:
Image 2020-10-26 at 11 48 04 AM

New FontAwesome icon:
Image 2020-10-26 at 11 48 16 AM

You can view all the icon libraries available with this plugin here:
https://react-icons.github.io/
For the sun icon, `RiSunLine` & `BiSun` closely match our current icon.

File: src/components/Icon.js (hunk 1 above)
Comment (howdyAnkit) [reply src/components/Icon.js:31]: I was actually struggling with the padding aspect ratio on the footer and with languages Icon when the size of Icons is small in Size. 
it should be safe to remove those now

**Comment by howdyAnkit** [comment]
Thank you so much @samajammin @ryancreatescopy  for your review. As I was sticking with Font awesome Icon as per the Issue. Totally on the same page with you @samajammin  on the alternatives I did find a few alternative Icons present on the mentioned React Icon website.

You can locate them on this Attached Link Below.
For Dark and Light Mode:-(https://react-icons.github.io/react-icons/icons?name=bi)
Tags:- BiAdjust(Dark Mode)  BiSun(Light Mode).

ss2

sun

Do let me know if there's any way I could help

**Review by samajammin — APPROVED** [review]
Thanks again for this @howdyAnkit! We may decide to go with an alternative to FontAwesome but this code is solid & we can make changes in a separate PR when the time comes.

**Comment by samajammin** [comment]
@all-contributors please add @howdyAnkit for code.

**Comment by howdyAnkit** [comment]
Thank you so much @samajammin for merging this. it really means a lot to me.

**Pull request merged**